packages/consts/src/consts.ts

```diff
@@ -75,139 +75,9 @@ export const META_ORIGINS = {
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-playwright-firefox:beta',
-        displayName: 'BETA: Node.js 14 + Playwright + Firefox on Debian',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-playwright-webkit:beta',
-        displayName: 'BETA: Node.js 14 + Playwright + WebKit on Debian',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-playwright:beta',
-        displayName: 'Node.js 14 + Playwright + All Browsers on Ubuntu',
-        copyChown: 'myuser:myuser',
-    },
-
-    // Deprecated:
-    // These are here because we made breaking changes in the client that could break existing single file actors.
-    // We will get a rid of this along with the whole single file logic.
-    {
-        name: 'apify/actor-node-basic',
-        displayName: '[DEPRECATED]: Node.js 12 on Alpine Linux',
-    },
-    {
-        name: 'apify/actor-node-chrome',
-        displayName: '[DEPRECATED]: Node.js 12 + Chrome on Debian',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-chrome-xvfb',
-        displayName: '[DEPRECATED]: Node.js 12 + Chrome + Xvfb on Debian',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-basic:beta',
-        displayName: '[DEPRECATED] BETA: Node.js 12 on Alpine Linux',
-    },
-    {
-        name: 'apify/actor-node-chrome:beta',
-        displayName: '[DEPRECATED] BETA: Node.js 12 + Chrome on Debian',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-chrome-xvfb:beta',
-        displayName: '[DEPRECATED] BETA: Node.js 12 + Chrome + Xvfb on Debian',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-basic:v0.21.10',
-        displayName: '[DEPRECATED]: Node.js 12 on Alpine Linux (Apify SDK v0.21.10)',
-    },
-    {
-        name: 'apify/actor-node-chrome:v0.21.10',
-        displayName: '[DEPRECATED]: Node.js 12 + Chrome on Debian (Apify SDK v0.21.10)',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-chrome-xvfb:v0.21.10',
-        displayName: '[DEPRECATED]: Node.js 12 + Chrome + Xvfb on Debian (Apify SDK v0.21.10)',
-        copyChown: 'myuser:myuser',
-    },
-    {
-        name: 'apify/actor-node-puppeteer',
-        displayName: '[DEPRECATED] Node.js 10 + Puppeteer on Debian - use apify/actor-node-chrome instead!',
-        copyChown: 'node:node',
-    },
-    {
-        name: 'apify/actor-node-puppeteer:beta',
-        displayName: '[DEPRECATED] BETA: Node.js 10 + Puppeteer on Debian - use apify/actor-node-chrome:beta instead!',
-        copyChown: 'node:node',
-    },
-];
-
-/**
- * Default image from ACTOR_BASE_DOCKER_IMAGES.
- */
-export const ACTOR_BASE_DOCKER_IMAGE_DEFAULT = ACTOR_BASE_DOCKER_IMAGES[0].name;
+export const ACTOR_BASE_DOCKER_IMAGE_DEFAULT = 'apify/actor-node';
```

Comment:
Maybe we can remove this const too? We have no notion of "default base Docker image" anymore, and it's only used in two files in integration tests, where we can/should hardcode it anyway.

Reply:
Yeah, right, I see it's only used on tests and there it can be hardcoded for sure.

Reply:
Done, fixed in https://github.com/apify/apify-core/pull/10579